### PR TITLE
Add personal keymap for Quefrency

### PR DESCRIPTION
**Pull request by hed854**
## Description

Adds a new keymap for Quefrency with backspace on the bottom row + Caps lock as a FN key + a arrow layer located on WSAD. This is loosely inspired from the KBParadise v60 keymap. It's also I2C-enabled.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/rev1/rev1.h [hunk 1]
```diff
@@ -101,3 +101,27 @@
     { RE1, RE2, RE9, RE4, RE5, RE6, RE7, RE8 }, \
     { KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, RA9, RB9 } \
   }
+
+#define LAYOUT_MONAKA( \
```
Comment (mechmerlin) [inline keyboards/keebio/quefrency/rev1/rev1.h]: We usually follow the LAYOUT_all_lower_case_description_here formatting. 

```suggestion
#define LAYOUT_monaka( \
```

File: keyboards/keebio/quefrency/keymaps/monaka/keymap.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT_MONAKA(
```
Comment (mechmerlin) [inline keyboards/keebio/quefrency/keymaps/monaka/keymap.c]:
```suggestion
  [_BASE] = LAYOUT_monaka(
```

File: keyboards/keebio/quefrency/keymaps/monaka/keymap.c [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_DEL,  KC_BSPC, \
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, \
+    MO(_FN1),KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT, \
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_UP, MO(_FN1), \
+    KC_LCTL, KC_LGUI, KC_LALT, KC_BSPC ,KC_SPC,           KC_SPC, KC_RALT, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+  ),
+
+  [_FN1] = LAYOUT(
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/monaka/keymap.c]: Shouldn't this also be: 
```suggestion
  [_FN1] = LAYOUT_monaka(
```

File: keyboards/keebio/quefrency/keymaps/monaka/keymap.c [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT_MONAKA(
+    KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_DEL,  KC_BSPC, \
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/keymaps/monaka/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

**Comment by drashna** [comment]
Any updates on this?

File: keyboards/keebio/quefrency/keymaps/monaka/keymap.c [hunk 5]
```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/keymaps/monaka/keymap.c]:
```suggestion
```

**Comment by hed854** [comment]
will fix and reply to all comments once I get a tad more free time to test on the board, sry for the delay

**Comment by drashna** [comment]
Any updates on this?

**Pull request closed without merging**